src/apify_client/clients/resource_clients/webhook.py

```diff
@@ -0,0 +1,116 @@
+from typing import Any, Dict, List, Optional
+
+from ..._utils import _snake_case_to_camel_case
+from ..base.resource_client import ResourceClient
+from .webhook_dispatch_collection import WebhookDispatchCollectionClient
+
+
+def _prepare_webhook_representation(
+    *,
+    event_types: Optional[List] = None,
+    request_url: Optional[str] = None,
+    payload_template: Optional[str] = None,
+    actor_id: Optional[str] = None,
+    actor_task_id: Optional[str] = None,
+    actor_run_id: Optional[str] = None,
+    ignore_ssl_errors: Optional[bool] = None,
+    do_not_retry: Optional[bool] = None,
+    idempotency_key: Optional[str] = None,
+    is_ad_hoc: Optional[bool] = None,
+) -> Dict:
+    """Prepare webhook dictionary representation for clients."""
+    webhook: Dict[str, Any] = {
+        _snake_case_to_camel_case(key): value
+        for key, value in locals().items() if value is not None and key not in ["actor_run_id", "actor_task_id", "actor_id"]
+    }
+
+    condition = {}
+
+    if actor_run_id is not None:
+        condition['actorRunId'] = actor_run_id
+        webhook['isAdHoc'] = True
+    elif actor_task_id is not None:
+        condition['actorTaskId'] = actor_task_id
+    elif actor_id is not None:
+        condition['actorId'] = actor_id
+
+    if condition != {}:
+        webhook["condition"] = condition
+
+    return webhook
+
+
+class WebhookClient(ResourceClient):
+    """Sub-client for manipulating a single webhook."""
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        """Initialize the WebhookClient."""
+        super().__init__(*args, resource_path='webhooks', **kwargs)
+
+    def get(self) -> Optional[Dict]:
+        """Retrieve the webhook.
+
+        https://docs.apify.com/api/v2#/reference/webhooks/webhook-object/get-webhook
+
+        Returns:
+            The retrieved webhook
+        """
+        return self._get()
+
+    def update(
+        self,
+        *,
+        event_types: Optional[List] = None,
+        request_url: Optional[str] = None,
+        payload_template: Optional[str] = None,
+        actor_id: Optional[str] = None,
+        actor_task_id: Optional[str] = None,
+        actor_run_id: Optional[str] = None,
+        ignore_ssl_errors: Optional[bool] = None,
+        do_not_retry: Optional[bool] = None,
+        is_ad_hoc: Optional[bool] = None,
+    ) -> Dict:
+        """Update the webhook.
+
+        https://docs.apify.com/api/v2#/reference/webhooks/webhook-object/update-webhook
+
+        Args:
+            event_types (list, optional): List of event types that should trigger the webhook.
+                                Present in the client constants as WebhookEventType. At least one is required.
```

Comment:
Couple of lines under this, you format overflowing text such as the description is aligned. Let's do it that way so we are consistent.